frontend/src/scenes/instance/AsyncMigrations/AsyncMigrations.tsx

```diff
@@ -0,0 +1,306 @@
+    component: AsyncMigrations,
+}
+
+const { TabPane } = Tabs
+
+export function AsyncMigrations(): JSX.Element {
+    const { user } = useValues(userLogic)
+    const { asyncMigrations, asyncMigrationsLoading, activeTab, asyncMigrationSettings } =
+        useValues(asyncMigrationsLogic)
+    const {
+        triggerMigration,
+        resumeMigration,
+        rollbackMigration,
+        forceStopMigration,
+        forceStopMigrationWithoutRollback,
+        loadAsyncMigrations,
+        loadAsyncMigrationErrors,
+        setActiveTab,
+    } = useActions(asyncMigrationsLogic)
+
+    const columns: LemonTableColumns<AsyncMigration> = [
+        {
+            title: 'Migration',
+            render: function Render(_, asyncMigration: AsyncMigration): JSX.Element {
+                const length = 40
+                const more = asyncMigration.description.length > length
+                const moreButton = (
+                    <a
+                        onClick={() => {
+                            Modal.info({
+                                title: `'${asyncMigration.name}' description`,
+                                content: <pre>{asyncMigration.description}</pre>,
+                                icon: <InfoCircleOutlined />,
+                                okText: 'Close',
+                                width: '80%',
+                            })
+                        }}
+                    >
+                        {` more`}
+                    </a>
+                )
+                return (
+                    <>
+                        {stringWithWBR(asyncMigration.name)}
+                        <span className="row-description">
+                            {asyncMigration.description.slice(0, length)}
+                            {more ? (
+                                <>
+                                    {`... `}
+                                    {moreButton}
+                                </>
+                            ) : null}
+                        </span>
+                    </>
+                )
+            },
+        },
+        {
+            title: 'Progress',
+            render: function Render(_, asyncMigration: AsyncMigration): JSX.Element {
+                const progress = asyncMigration.progress
+                return (
+                    <div>
+                        <Progress percent={progress} />
+                    </div>
+                )
+            },
+        },
+        {
+            title: 'Status',
+            render: function Render(_, asyncMigration: AsyncMigration): JSX.Element {
+                const status = asyncMigration.status
+                let type = 'default'
+                if (status === AsyncMigrationStatus.Errored) {type = 'danger'}
+                if (status === AsyncMigrationStatus.CompletedSuccessfully) {type = 'success'}
+                if (status === AsyncMigrationStatus.Running) {type = 'warning'}
```

Comment:
these don't seem to work, but I'm not sure why?
<img width="250" alt="Screen Shot 2022-01-24 at 16 32 05" src="https://user-images.githubusercontent.com/890921/150813599-9ac6a1e0-9988-4176-9346-541de8e21b67.png">

Reply:
because of our css file that had defined .danger etc to use those colors, i.e. we were trying to write on red with red